### DISCUSSION TITLE
Lesson-7 | Error: Internal EVM error during simulation

### DISCUSSION THREAD

**Discussion by vigneshbalu96** [Cyfrin/foundry-full-course-cu#2329] [General]
while running the 'script' command using local 'anvil', it gives error:

```solidity
$ forge script script/Interactions.s.sol:FundFundMe --rpc-url http://127.0.0.1:8545
```
[⠢] Compiling...
[⠑] Compiling 3 files with Solc 0.8.19
[⠘] Solc 0.8.19 finished in 3.45s
Compiler run successful!
EIP-3855 is not supported in one or more of the RPCs used.
Unsupported Chain IDs: 31337.
Contracts deployed with a Solidity version equal or higher than 0.8.20 might not work properly.
For more information, please see https://eips.ethereum.org/EIPS/eip-3855
Script ran successfully.

== Logs ==
  Funded FundMe with 10000000000000000

## Setting up 1 EVM.
Error: 
🚨 **Internal EVM error during simulation**

Context:
- Error #0: backend: failed while inspecting
- Error 1: transaction validation error: lack of funds (0) for max fee (10000000000000000)
- Error 2: lack of funds (0) for max fee (10000000000000000)

Thanks in Advance!

**Comment by EngrPips** [comment]
Hello @vigneshbalu96, This seems like a simulation error, `Internal EVM error during simulation.` Do you know if the script does all it was supposed to do?

**Reply by vigneshbalu96** [reply]
Yeah, but the issue is might be due to the chain Id as the anvil default chain Id is 31337? It also given the warning as 
'Unsupported Chain IDs: 31337' as the above output. @EngrPips

**Reply by EngrPips** [reply]
I see, I am not sure why it is giving that warning, but I assume it is not a big deal since the script worked perfectly.

**Reply by vigneshbalu96** [reply]
I also ensured that the version is 'forge 0.2.0' and lowered the version above 0.8.20 to below 0.8.20 (0.8.19 as recommended), giving the same error.

**Reply by EngrPips** [reply]
Can you check on the official Foundry repo if there are any similar issue reported?

**Reply by vigneshbalu96** [reply]
No, I didn't find any. Any help could be helpful! Thanks.

**Reply by EngrPips** [reply]
I am not sure what could cause this maybe update foundry, run `foundryup`, then `forge clean` then retry the process.

**Reply by usgeeus** [reply]
How did you solve this?

**Reply by EngrPips** [reply]
> How did you solve this?

You can maybe report on the official Foundry repo and see what the team has to say.
